docs/docs/guides/integrate/identity-providers/_ldap_warn.mdx

```diff
@@ -0,0 +1,4 @@
+:::caution
+LDAP does send your user passwords in clear text. Due to this we do not recommend using it with the ZITADEL cloud.
+If you want to use it anyway, please reach out to us.
```

Comment:
```suggestion
LDAP does send your passwords in clear text. Due to this we do not recommend using it with the ZITADEL cloud.
If you want to use it anyway, please reach out to us.
```

Reply:
Maybe we should quickly elaborate ldap vs ldaps

Reply:
Do we support ldaps or just STARTTLS?

Reply:
What about this? https://github.com/zitadel/zitadel/pull/5677/commits/d0d7cc6cc2a6feff7ba128d3148b7b3ecbd6116a